tests.toml

```diff
@@ -7,3 +7,11 @@ test_format = 1.0
     # ------------
     # Tests to run
     # ------------
+
+    exclude = ["install.subdir"]
```

Comment:
```suggestion
```

No need to exclude subdir install, CI will already not run subdir install because there's no path question